examples/example_fastapi/main.py

```diff
@@ -44,15 +45,23 @@
     allow_methods=["*"],
     allow_headers=["*"],
 )
-cache = VmCache()
+
+# Initialize cache on startup event to avoid running loop errors
+cache: VmCache | None = None
```

Comment:
What was the problem with the previous init method? This introduces a typing PITA that `cache` can be `None` on every access.